src/briefcase/commands/base.py

```diff
@@ -809,6 +809,10 @@ def update_cookiecutter_cache(self, template: str, branch="master"):
                 # Template cache path exists, but isn't a git repository
                 # Just use the template directly, rather than attempting an update.
                 cached_template = template
+            except ValueError:
+                raise BriefcaseCommandError(
+                    f"Git repository in a weird state, delete {cached_template} and try briefcase create again"
+                )
```

Comment:
`from e` pls....wonder if there's a linter for this
```suggestion
            except ValueError as e:
                raise BriefcaseCommandError(
                    f"Git repository in a weird state, delete {cached_template} and try briefcase create again"
                ) from e
```

Reply:
Agreed that an automated check for this would be nice - not sure what that would look like in terms of implementation, though...